src/toTransform.js

```diff
@@ -101,27 +101,57 @@
   function convertOffsetAnchorPosition (properties, element) {
     // According to spec: https://drafts.fxtf.org/motion-1/#offset-anchor-property
     // If offset-anchor is set to auto then it will compute to the value of offset-position.
-    var position = 'auto';
-
     if (element === undefined) {
       return null;
     }
 
+    var position = 'auto';
     if ('offsetPosition' in properties) {
       position = internalScope.offsetPositionAnchorParse(properties['offsetPosition']);
     }
 
-    if (position === 'auto' || position === undefined || position === null) {
-      return null;
-    }
-
     var anchor = 'auto';
     if ('offsetAnchor' in properties) {
       anchor = internalScope.offsetPositionAnchorParse(properties['offsetAnchor']);
     }
 
+    var transformOrigin = window.getComputedStyle(element).transformOrigin;
+    transformOrigin = transformOrigin.split(/\s+/);
+    transformOrigin = transformOrigin.map(internalScope.offsetDistanceParse);
+
     if (anchor === 'auto' || anchor === undefined || anchor === null) {
-      anchor = position;
+      if ('offsetPath' in properties && properties['offsetPath'] === 'none') {
```

Comment:
Perhaps also allow properties['offsetPath'] to be null / undefined?
  (!properties['offsetPath'] || properties['offsetPath'] === 'none')